wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,258 @@
 
--   **Tags:** **Keywords to help in searches.** Common uses include album names, guest difficulty creators, company name and other terms that may be used by people when searching for specific wants (like eroge or visual novel for example if your beatmap's song is based on an eroge visual novel). If your map has "collab" elements mapped by other mappers, their names can go here. Tags are separated by spaces. It is not compulsory to fill in but is greatly appreciated when using the Web search and osu! search in-game.
-    -   Note: If you have a tag of "qwertyuiop" as a tag on your beatmap, you can also find your beatmap in-game by typing in "qwertyuiop".
+This field can also be used to indicate that it is a guest difficulty (e.g. "Larto's Hard").
 
-Again, to avoid issues, please make sure that everything (aside from Difficulty name) is filled out identically for each difficulty.
+#### Source
 
-Difficulty
-----------
+**The origin of the song.** Generally, this field should be used only for anime or video game songs and occasionally for novelty (movies, TV, or internet) songs. If the song is from an anime, a video game, or is famous for its use in a TV or movie theme; then the title of that source should be added in this field. If otherwise, this field should be left empty.
 
-![Difficulty section](img/SS_Difficulty_Settings.jpg "Difficulty section")
+*Notice: This is not to be used for album titles, those belong in the [tags](#tags) field.*
 
-Tab related to the core of beatmap's settings. Each setting affects different aspects of the beatmap.
+Not every song needs something on the source field. The source is only for songs that are from something (e.g. a video game or an anime show). Regular rock and pop songs can have this field be left empty. Other relevant information should be entered on the [tags](#tags) field such as album title or subgenre information.
+
+#### Tags
+
+**A list of keywords to help with search queries.** Commonly, tags should include album names, guest creator usernames, company names, and other terms that may be used by people when searching for beatmaps. If the beatmap has collaboration elements (such as storyboarding) done by other users, their usernames should be added to this field.
+
+Tags are separated with spaces (`Space`). It is not necessary to use this field as other metadata from the other fields are used to help query beatmaps; however, it is greatly appreciated for users trying to find your beatmaps when using the web search bar or the in-game search bar.
+
+## Difficulty
+
+![Two screenshots of the difficulty tab. osu!mania varies from the other game modes.](img/SS_Difficulty.jpg "Difficulty section. The right section is for osu!mania-specific beatmap.")
+
+*Notice: The game mode specific [Ranking Criterias](/wiki/Ranking_Criteria) has rankable values that should be used.*
+
+The beatmap's difficulty settings. Each setting affects different aspects of the beatmap.
```

Comment:
Either make the first sentence complete or bold it like the `Song and Map Metadata` settings.